more Meta area fixes

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
fixes misc area errors around the map
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
mapping errors = bad
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![image](https://user-images.githubusercontent.com/79304582/193687304-8dede624-12f5-4738-879d-e844e4a0494b.png)

![image](https://user-images.githubusercontent.com/79304582/193687343-eddfe7c5-89f5-4981-a44c-080d5db72e34.png)

![image](https://user-images.githubusercontent.com/79304582/193687380-85f741c5-d430-4489-a559-9ebc6e7a2bf5.png)

![image](https://user-images.githubusercontent.com/79304582/193687409-79759269-aab9-4601-ad4e-5b8ab3ef50dc.png)

![image](https://user-images.githubusercontent.com/79304582/193687442-139c16f4-d626-4527-bf9d-fe4ac5f05230.png)

![image](https://user-images.githubusercontent.com/79304582/193687465-ec582de8-3d35-41cd-8d46-5dc7429b2134.png)


</details>

## Changelog
:cl:
fix: fixed a few zone errors on Meta station
del: Removes redundant wiring on Meta
add: adds missing wiring on Meta
del: removes redundant piping on Meta
add: adds missing piping on Meta
del: removes redundant disposals on Meta
del: removes hidden wall lattices on Meta
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
more meta zone fixes

spockye committed:
sci dock zone

PestoVerde322 reviewed (dismissed):
Everything good to go, approved.

spockye committed:
Update MetaStation.dmm

Removes redundant wiring
adds missing wiring
removes redundant piping
adds missing piping
removes redundant disposals
removes hidden wall lattices
area fixes

spockye committed:
Merge remote-tracking branch 'upstream/master' into meta-Zone-fixes-
